android/assets/jsons/translations/Traditional_Chinese.properties

```
@@ -1804,15 +1804,15 @@ Granary = 糧倉
 Temple of Artemis = 阿爾忒彌斯神廟
 'It is not so much for its beauty that the forest makes a claim upon men's hearts, as for that subtle something, that quality of air, that emanation from old trees, that so wonderfully changes and renews a weary spirit.' - Robert Louis Stevenson = 森林之所以能打動人們的心，與其說是它的美麗，不如說是因為它微妙的事物，那種空氣的清新，那種從古樹中散發出來的東西，它如此奇妙地改變和撫慰了疲憊的精神。 ——羅伯特·路易斯·史蒂文森
  # Requires translation!
-[relativeAmount]% [stat] [cityFilter] = 
+[relativeAmount]% [stat] [cityFilter] = [cityFilter][relativeAmount]%[stat]
 [relativeAmount]% Production when constructing [baseUnitFilter] units [cityFilter] = [cityFilter]建造[baseUnitFilter]單位時[relativeAmount]%產能
 
 The Great Lighthouse = 大燈塔
 'They that go down to the sea in ships, that do business in great waters; these see the works of the Lord, and his wonders in the deep.' - The Bible, Psalms 107:23-24 = “在海上坐船，在大水中經理事務的；他們看見耶和華的作為，並他在深水中的奇事。”——聖經·詩篇 107:23-24
  # Requires translation!
-for [mapUnitFilter] units = 
+for [mapUnitFilter] units = 對於[mapunitfilter]單位
```

```suggestion
for [mapUnitFilter] units = 對於[mapUnitFilter]單位
```
Find the cause of failed check!

---

Thank you very much. I didn't even notice.